server/integration/testsuite/src/test/java/org/infinispan/server/test/util/security/SaslConfigurationBuilder.java

```diff
@@ -20,23 +18,14 @@
  */
 public class SaslConfigurationBuilder extends ConfigurationBuilder {
 
-   protected static final String DEFAULT_TEST_REALM = "ApplicationRealm";
-
-   protected static final String DEFAULT_KEYSTORE_PATH = ITestUtils.SERVER_CONFIG_DIR + File.separator
-         + "keystore_client.jks";
-   protected static final String DEFAULT_KEYSTORE_PASSWORD = "secret";
-   protected static final String DEFAULT_TRUSTSTORE_PATH = ITestUtils.SERVER_CONFIG_DIR + File.separator
-         + "truststore_client.jks";
-   protected static final String DEFAULT_TRUSTSTORE_PASSWORD = "secret";
-
    private final String saslMech;
 
    public SaslConfigurationBuilder(String saslMech) {
       this.saslMech = saslMech;
    }
 
    public SaslConfigurationBuilder forCredentials(String login, String password) {
-      this.security().authentication().callbackHandler(new SimpleLoginHandler(login, password, DEFAULT_TEST_REALM));
+      this.security().authentication().callbackHandler(new SimpleLoginHandler(login, password, Constants.DEFAULT_TEST_REALM));
```

Comment:
`Constants` is a way too generic name, have a look at https://github.com/infinispan/infinispan/blob/master/server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/Constants.java#L57 to see what happens when you name a class like that :)

You can leave the constants here, and just make them public.